test/post-api-score-test.js

```diff
@@ -29,30 +39,56 @@ buster.testCase('postApiScore', {
     buster.assert.calledWith(this.create, {})
   },
 
-  'should wrap result in object with success flag set to true': function () {
+  /* 'should return failure code on empty request': function (done) {
```

Comment:
Auskommentierten Code entweder entfernen, wenn er nicht mehr relevant ist, oder, wenn es unbedingt sein muss den Test ausschalten mit '// should ...'.
Warum ist der auskommentiert?